frontend/src/reducer/currentWorkspace.js

```diff
@@ -160,7 +160,7 @@ export default function currentWorkspace (state = defaultWorkspace, action) {
         ...state,
         contentReadStatusList: state.contentReadStatusList.filter(id => id !== action.unreadId),
         recentActivityList: [
-          state.recentActivityList.find(content => content.id === action.unreadId),
+          ...(state.recentActivityList.find(content => content.id === action.unreadId) || []),
```

Comment:
Yes true, this line was here to prevent `state.recentActivityList.find(content => content.id === action.unreadId` -> null, in order to prevent this I suggest to use `...state.recentActivityList.filter(content => content.id === action.unreadId)` so it will return [] if it can not find it